conf/gulp-tasks/library.gulpfile.js

```diff
@@ -21,9 +21,21 @@ const uglify = require('gulp-uglify-es').default;
 const NAMESPACE = 'ANSWERS';
 
 function getLibVersion () {
-  return require('child_process')
-    .execSync('git describe --tags')
-    .toString().trim();
+  try {
+    let insideWorkTree = require('child_process')
+      .execSync('git rev-parse --is-inside-work-tree 2>/dev/null')
+      .toString().trim();
+    if (insideWorkTree) {
+      return require('child_process')
+        .execSync('git describe --tags')
+        .toString().trim();
+    }
+  } catch (e) {
```

Comment:
Which of the commands above can cause the exception?

Reply:
Both of the commands can cause an exception. The latter, I have only seen an exception when we are not in a git repository, so that case should be caught by the first command.